action.yaml

```diff
@@ -61,6 +61,10 @@ inputs:
     description: Fail the CI if Cimon encountered an error
     required: false
     default: "false"
+  upload-artifact:
```

Comment:
Can you rename it to `report-artifact-log` and put it after `report-process-tree` ?